src/Relation/Binary/Apartness.agda

```diff
@@ -0,0 +1,68 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Apartness relations
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Binary.Apartness where
```

Comment:
Unless I'm mistaken, there's not a deep hierarchy of apartness relations? If that's the case then I think we should just fold the definitions into the correct places in the standard `Relation.Binary` modules. i.e.

- `Comparison`/`Tight` in `Relation.Binary.Definitions`
- `sym⇒sym-¬`/`comp⇒trans-¬` etc. in `Relation.Binary.Consequences`
- `IsApartness` in `Relation.Binary.Structures`
- `Apartness` in `Relation.Binary.Bundles`
- `¬#-equiv` in `Relation.Binary.Properties.Apartness`

Reply:
These have now been implemented, with some renaming to make them a bit more general (they're not all specific to apartness relations).